KBH.INC

```diff
@@ -0,0 +1,6 @@
+GLOBAL install_keyboard_handler:PROC            ;; void install_keyboard_handler()
+GLOBAL uninstall_keyboard_handler:PROC          ;; void uninstall_keyboard_handler()
+
+GLOBAL __keyb_keyboardState:BYTE:128
```

Comment:
The __keyb* variables shouldn't be global afaiu.

Reply:
Needed to see which keys were pressed